TEST_WORKFLOW.md

```diff
@@ -0,0 +1,136 @@
+X-Road Tests development
+# Workflow Policy
+
+v1.0
+
+## 1	General
+
+1.1	This document establishes workflow policy for joint development of X-Road software. Workflow Policy strives to serve as a handbook for all practical aspects of development.
```

Comment:
This document establishes workflow policy for joint development of X-Road software _test development_.

Reply:
done